module/applications/actor/group-sheet.mjs

```diff
@@ -50,6 +59,11 @@ export default class GroupActorSheet extends ActorSheet {
     // Inventory
     context.itemContext = {};
     context.inventory = this.#prepareInventory(context);
+    context.expandedData = {};
+    for ( const id of this._expanded ) {
+      const item = this.actor.items.get(id);
+      if ( item ) context.expandedData[id] = await item.getChatData({secrets: this.actor.isOwner});
```

Comment:
Looks like this might conflict with changes in #2206

Reply:
That was on the base sheet, this is on the group sheet.